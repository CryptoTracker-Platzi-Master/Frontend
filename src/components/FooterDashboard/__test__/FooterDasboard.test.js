import React from "react";
import "@testing-library/jest-dom/extend-expect";
import { render } from "@testing-library/react";
import { FooterDashboard } from "../";

describe("[ component / FooterDashboard ]", () => {
  describe("when  `FooterDashboard` is mounted", () => {
    it("[`FooterDashboard`]", () => {
      const { asFragment } = render(<FooterDashboard />);
      expect(asFragment()).toMatchSnapshot();
    });
  });
});
