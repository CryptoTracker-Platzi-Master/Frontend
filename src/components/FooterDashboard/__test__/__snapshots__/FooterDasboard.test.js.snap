// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[ component / FooterDashboard ] when  \`FooterDashboard\` is mounted [\`FooterDashboard\`] 1`] = `
<DocumentFragment>
  <footer
    class="footer-dashboard"
  >
    <figure
      class="footer-dashboard__container-img"
    >
      <img
        alt="Imagen de logotipo"
        class="footer-dashboard__container-img--img"
        src="logoTipo.png"
      />
    </figure>
  </footer>
</DocumentFragment>
`;
